projects/demo-integrations/cypress/support/snapshot.ts

```diff
@@ -0,0 +1,46 @@
+import {WAIT_BEFORE_SCREENSHOT} from './shared.entities';
+
+export interface TuiSnapshotOptions {
+    wait?: number;
+    prefixName?: string;
+    name?: string;
+    postfixName?: string;
+    capture?: 'viewport';
+}
+
+export const tuiSnapshot = (
```

Comment:
Move all these logic inside
https://github.com/Tinkoff/taiga-ui/blob/main/projects/testing/cypress/snapshot/command.ts#L5

```ts
export function tuiAddMatchImageSnapshotCommand(
    options: Options & TuiSnapshotCommandOptions,
): void {
    const matchSnapshotFn = matchImageSnapshotCommand(options);

    Cypress.Commands.add(
        'matchImageSnapshot',
        {
            prevSubject: ['optional', 'element', 'window', 'document'],
        },
        (prevSubject, name: string, options?: Options & TuiSnapshotCommandOptions) => {
            
            // any custom logic here

            matchSnapshotFn(prevSubject, name, options);
        },
    );
}
```

Reply:
See example 
https://github.com/Tinkoff/taiga-ui/pull/1721/files#diff-224d94fd4507d13303205b318fd60c5d0a03b53dc1629ee75eb4ea4ba2fc17ebR27

Let's create command `tuiWaitAllImages` and use before `matchSnapshotFn`

Reply:
fixed